packages/neuron-wallet/src/controllers/export-debug.ts

```diff
@@ -88,6 +89,32 @@ export default class ExportDebugController {
     })
   }
 
+  private addBundledCKBLog = () => {
+    const name = 'bundled-ckb-log.json'
+    const SIZE_TO_READ = 32_000
+
+    return new Promise((resolve, reject) => {
+      const logPath = path.resolve(ckbPath(), 'data', 'logs', 'run.log')
+      if (!fs.existsSync(logPath)) { reject(new Error("File not found")) }
```

Comment:
Should it stop processing for the rest of this promise if the file doesn't exist?
For example: `return reject(new Error("File not found"))`